code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1255,16 +1255,55 @@
 	else
 		germ_level += n
 
+/mob/living/carbon/human/proc/check_and_regenerate_organs(var/mob/living/carbon/human/H) //Regenerates missing limbs/organs.
+	var/datum/species/S = H.species
+	var/list/species_organs = S.has_organ
+	var/list/species_limbs = S.has_limbs

```

Comment:
Going to remove this and the above two lines as they're unnecessary since H.species.has_organ and H.species_has_limbs can be used instead, commit pending